.github/workflows/package.yaml

```diff
@@ -108,42 +109,49 @@ jobs:
         name: Rancher Desktop-win.zip
         path: dist/Rancher Desktop-*-win.zip
         if-no-files-found: error
-    - uses: actions/upload-artifact@v2
-      if: startsWith(matrix.os, 'ubuntu-')
+    - name: set zip_name env var
+      run: |
+        # in pull requests GITHUB_REF_NAME is in the form "<pr_number>/merge";
+        # remove slashes since they aren't valid in filenames
+        no_slash_ref_name=$(echo $GITHUB_REF_NAME | sed -E 's/\//-/g')
+        zip_name="rancher-desktop-linux-${no_slash_ref_name}.zip"
+        echo "zip_name=$zip_name" >> $GITHUB_ENV
+    - name: Upload zip as workflow artifact
+      uses: actions/upload-artifact@v2
+      if: startsWith(matrix.os, 'ubuntu-') && github.ref_type == 'branch'
       with:
-        name: Rancher Desktop-linux.zip
+        name: ${{ env.zip_name }}
```

Comment:
FYI, `zip_name` (formerly `Rancher Desktop-linux.zip`) is a zip containing the zip uploaded to the release. Go download it from a packaged actions run like https://github.com/rancher-sandbox/rancher-desktop/actions/runs/1725878053. The linux build from the action for the tag is something we download, unzip, and then take the zip inside and upload to the release.

Reply:
Yes, this was something I realized while putting this together. I wonder if we should document this process somewhere. Though it will probably change at some point anyways.